admin_manual/release_notes.rst

```diff
@@ -0,0 +1,266 @@
+app.
+
+Changes to File Previews
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+For security and performance reasons, file previews are available only for 
+image files, covers of MP3 files, and text files, and have been disabled for 
+all other filetypes. Files without previews are represented by generic icons 
+according to their file types. 
+
+4GB Limit on SFTP Transfers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Because of limitations in ``phpseclib``, you cannot upload files larger than 
+4GB over SFTP.
+
+"Not Enough Space Available" on File Upload
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Setting user quotas to ``unlimited`` on an ownCloud installation that has 
+unreliable free disk space reporting-- for example, on a shared hosting 
+provider-- may cause file uploads to fail with a "Not Enough Space Available" 
+error. A workaround is to set file quotas for all users instead of 
+``unlimited``.
+
+No More Expiration Date On Local Shares
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In older versions of ownCloud, you could set an expiration date on both local 
+and public shares. Now you can set an expiration date only on public shares, 
+and 
+local shares do not expire when public shares expire.
+
+Zero Quota Not Read-Only
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Setting a user's storage quota should be the equivalent of read-only, however, 
+users can 
+still create empty files.
+
+=================
+Enterprise 7 Only
+=================
+
+Windows Network Drive
+^^^^^^^^^^^^^^^^^^^^^
+Windows Network Drive runs only on Linux servers because it requires the Samba 
+client, which is included in all Linux distributions. 
+
+``php5-libsmbclient`` is also required, and there may be issues with older 
+versions of ``libsmbclient``; see Using External Storage > Installing and 
+Configuring the Windows Network Drive App in the Enterprise Admin manual for 
+more information. 
+
+By default CentOS has activated SELinux, and the ``httpd`` process can not make 
+outgoing network connections. This will cause problems with curl, ldap and samba 
+libraries. Again, see Using External Storage > Installing and Configuring the 
+Windows Network Drive App in the Enterprise Admin manual for instructions.
+
+Sharepoint Drive SSL
+^^^^^^^^^^^^^^^^^^^^
+
+The SharePoint Drive app does not verify the SSL certificate of the SharePoint 
+server or the ownCloud server, as it is expected that both devices are in the 
+same trusted environment.
+
+Shibboleth and WebDAV Incompatible
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Shibboleth and standard WebDAV are incompatible, and cannot be used together in 
+ownCloud. If Shibboleth is enabled, the ownCloud client uses an extended WebDAV 
+protocol
+
+No SQLite
+^^^^^^^^^
+
+SQLite is no longer an installation option for ownCloud Enterprise Edition, as 
+it not suitable for multiple-user installations or managing large numbers of 
+files.
+
+No App Store
+^^^^^^^^^^^^
+
+The App Store is disabled for the Enterprise Edition.
```

Comment:
Then it's indeed enabled in 8EE but might void support contract?

@MTRichards Clarification pls